modules/account-lib/src/coin/baseCoin/ed25519KeyPair.ts

```diff
@@ -1,22 +1,98 @@
+import * as nacl from 'tweetnacl';
+import * as hex from '@stablelib/hex';
+import * as stellar from 'stellar-sdk';
+import { Seed, generateAccountFromSeed, generateAccount } from 'algosdk';
 import { BaseKeyPair } from './baseKeyPair';
 import { AddressFormat } from './enum';
 import { NotImplementedError } from './errors';
+import { isPrivateKey, isPublicKey, isSeed, DefaultKeys, KeyPairOptions, ByteKeys } from './iface';
+
+const DEFAULT_SEED_SIZE_BYTES = 32;
```

Comment:
hmm can we include a reference here back to the hedera sdk

Reply:
it doesn't have to be imported, just a comment saying where this is coming from and what line